blockchain/standalone/tx.go

```diff
@@ -125,6 +125,7 @@ func IsTreasuryBase(tx *wire.MsgTx) bool {
 	if len(tx.TxOut[1].PkScript) != 14 ||
 		tx.TxOut[1].PkScript[0] != opReturn ||
 		tx.TxOut[1].PkScript[1] != opData12 {
+
```

Comment:
Looks like an accidental newline added here

Reply:
This was intentional as well.  In general, whenever there is a multiline if statement, there is a blank line to separate the code path from it, because it is otherwise indented at the exact same depth and can be hard to distinguish.

Reply:
Makes sense, the multiline if statement right above this one could be adjusted too for consistency then

Reply:
Good point.  Updated to to put both conditions on the same line since I noticed it fits.